api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -186,12 +193,13 @@
      * is equivalent to infinity; The operator hence will be attempted to be recovered indefinitely unless this value
      * is set to anything else.
      */
-    Attribute<Integer> RECOVERY_ATTEMPTS = new Attribute<Integer>(new Integer2String());
+    Attribute<Integer> RECOVERY_ATTEMPTS = new Attribute<>(new Integer2String());
```

Comment:
IMHO, we should just change <...> to <> in all relevant places.  Line wrap should not be the main factor of doing this.  We should be consistent and that's one of the main reasons why the code style change is needed.